modules/kirillov_m_quadratic_solution/test/test_quadratic_solution.cpp

```diff
@@ -0,0 +1,122 @@
+// Copyright 2024 Kirillov Maxim
+
+#include <gtest/gtest.h>
+#include <unordered_set>
+#include "../include/quadratic_solution.h"
+
+TEST(Kirillov_M_Quadratic_Solution, TestInitCoefficients) {
+    double a = 1, b = 2, c = 1;
+    QuadraticSolver solver(a, b, c);
+    EXPECT_EQ(solver.getA(), a);
+    EXPECT_EQ(solver.getB(), b);
+    EXPECT_EQ(solver.getC(), c);
+}
+
+TEST(Kirillov_M_Quadratic_Solution, TestDiscriminantZero) {
+    double a = 1, b = 2, c = 1;
+    QuadraticSolver solver(a, b, c);
+    EXPECT_EQ(solver.getDiscriminant(), 0);
+}
+
+TEST(Kirillov_M_Quadratic_Solution, TestDiscriminantPositive) {
+    double a = 2.4, b = 3, c = -1.9;
+    QuadraticSolver solver(a, b, c);
+    EXPECT_GT(solver.getDiscriminant(), 0);
+}
+
+TEST(Kirillov_M_Quadratic_Solution, TestDiscriminantNegative) {
+    double a = 2, b = -3.3, c = 5.13;
+    QuadraticSolver solver(a, b, c);
+    EXPECT_LT(solver.getDiscriminant(), 0);
+}
+
+TEST(Kirillov_M_Quadratic_Solution, TestGetRealRoots) {
+    double a = 1, b = -5, c = 6;
+    QuadraticSolver solver(a, b, c);
+    auto solution = solver.solveRealRoots();
+    EXPECT_TRUE(solution.first);
+    std::unordered_set<double> expectedRoots = {2, 3};
```

Comment:
```suggestion
    std::unordered_set<double> expectedRoots{2, 3};
```
Также поправьте пожалуйста в других местах